pkg/mcs/lighthouse.go

```diff
@@ -0,0 +1,91 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mcs
+
+import (
+	"errors"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	lhconst "github.com/submariner-io/lighthouse/pkg/constants"
+)
+
+// Lighthouse provides a wrapper around Submariner's Lighthouse naming
+// conventions. The type is not strictly needed (hence the lack of receiver
+// object in all calls) but can be used a a scope for shorter function
+// names.
+type Lighthouse struct{}
```

Comment:
I would call it LightHouseUtils at least
Acutally I didn't understand why its better that a package named `lhutils` with static functions.

Reply:
It's not better than a `lhutils` package other than avoid proliferation of packages (I wanted to add it to `mcs` so our affected code base is limited).
I could have named the functions with a `Lighthouse` prefix as well. Having a struct just means you can use a shorter name for it. A package would achieve the same (using a shorter import name). 
I can remove this struct altogether if not consistent or style us unexpected.

Reply:
moved to new package `lhutil`